File: Boucles/TP4_7.js
function oddEven(array) {
  let i = 0;
  let counter=0;
  while (i < (array.length)) {
    let currentElement = array[i];
    if (currentElement % 2 == 0) {
      counter++;
    } else {}
    i++;
  }
  console.log(counter);
}

let test = [1,2,3,4,5,6,7];
oddEven(test);

File: Boucles/TP4_7.py
def oddEven(array) :
    i = 0;
    counter=0;
    while i < len(array):
        currentElement = array[i];
        if (currentElement % 2 == 0):
            counter+=1;
        i+=1;
    return counter


test = [1,2,3,4,5,6,7,8];
result = oddEven(test);
print(result)
